server/migrations/20190305132457-create-user.js

```diff
@@ -22,6 +22,9 @@ module.exports = {
       verified: {
         type: Sequelize.BOOLEAN
       },
+      socialProvider: {
+        type: Sequelize.STRING
```

Comment:
**[comment]** An ENUM data type could be useful here, considering that we're using Postgres